### PR TITLE
Big/Vector Icons

### PR DESCRIPTION
**Pull request by Laserlicht**
adresses https://github.com/vcmi/vcmi/issues/1993

also contains svg and derives all other logos from it

### CODE REVIEW

**Comment by Laserlicht** [comment]
The vcmi.icns for Mac OS is missing. I currently don't now how to create from svg.

**Comment by IvanSavenko** [comment]
Slightly different than what we had before, but I think I actually like the change

**Comment by Laserlicht** [comment]
> Slightly different than what we had before, but I think I actually like the change

Yes - an exact vector rebuild wasn't easy. The biggest change is the removal of the blur for the big icons.

File: client/icons/generate_icns.py [hunk 1]
```diff
@@ -15,4 +15,3 @@
 
 os.system("iconutil -c icns vcmi.iconset")
 shutil.rmtree("vcmi.iconset")
-
```
Comment (kambala-decapitator) [inline client/icons/generate_icns.py:18]: please drop this change

**Comment by kambala-decapitator** [comment]
wanted to create icns, but looks like the script is broken (requires py2?)

```
> cd vcmi/osx
> chmod +x ../client/icons/generate_icns.py
> ../client/icons/generate_icns.py ~/Desktop/vcmiclient.1024x1024.png
from: can't read /var/mail/PIL
Version: ImageMagick 7.1.0-61 Q16-HDRI x86_64 20866 https://imagemagick.org
...
import: delegate library support not built-in '' (X11) @ error/import.c/ImportImageCommand/1297.
/Users/kambala/dev/vcmi/vcmi/client/icons/generate_icns.py: line 4: syntax error near unexpected token `('
/Users/kambala/dev/vcmi/vcmi/client/icons/generate_icns.py: line 4: `img = Image.open(sys.argv[1])'
```

**Comment by Laserlicht** [comment]
> wanted to create icns, but looks like the script is broken (requires py2?)

I was able to create this file with the linux package `icnsutils`

and then with `png2icns vcmi.icns vcmiclient.*.png` (images (e.g. vcmiclient.16x16.png) with 16, 32, 48, 128, 256, 512, 1024 in folder)

**Comment by kambala-decapitator** [comment]
thanks!

checked the file: looks good!

**Comment by IvanSavenko** [comment]
By the way, @Laserlicht is it possible to create separate svg's that contain:
1) only shield, without text
2) only styled text, without shield?

Obviously optional, but we had such request some time ago, and since you're at least somewhat familiar with svg perhaps you're willing to do this?

**Comment by Laserlicht** [comment]
@IvanSavenko With SVG it's a simple task.
vcmiclient_shield
vcmiclient_text

**Review by IvanSavenko — APPROVED** [review]

**Review by Nordsoft91 — APPROVED** [review]

**Pull request merged**